webapp/src/test/java/com/box/l10n/mojito/service/screenshot/ScreenshotServiceTest.java

```diff
@@ -134,9 +134,8 @@ public void testDeleteScreenshotNoThirdPartyConfig() throws RepositoryNameAlread
     screenshotService.thirdPartyService = thirdPartyServiceMock;
 
     Screenshot screenshot = new Screenshot();
-    screenshot.setId(1L);
```

Comment:
See Hibernate bug: https://hibernate.atlassian.net/browse/HHH-17634
And this is the PR: https://github.com/hibernate/hibernate-orm/pull/7698